app/webpacker/components/ScrambleMatcher/index.jsx

```diff
@@ -61,6 +61,22 @@ function ScrambleMatcher({ wcifEvents, competitionId, initialScrambleFiles }) {
     mutationFn: () => submitMatchedScrambles(competitionId, matchState),
   });
 
+  const roundIds = useMemo(() => _.flatMap(wcifEvents, 'rounds').map((r) => r.id), [wcifEvents]);
```

Comment:
I'm a bit partial to using JS stdlib wherever possible
```suggestion
  const roundIds = useMemo(() => wcifEvents.flatMap((event) => event.rounds).map((r) => r.id), [wcifEvents]);
```